### DISCUSSION TITLE
creating-a-custom-xunit-theory-test-dataattribute-to-load-data-from-json-files/

### DISCUSSION THREAD

**Discussion by andrewlock** [andrewlock/blog-comments#237] [Announcements]
# creating-a-custom-xunit-theory-test-dataattribute-to-load-data-from-json-files/

In this post I'll show how you can create a custom implementation of an xUnit DataAttribute that loads the data for theory tests from JSON files.

https://andrewlock.net/creating-a-custom-xunit-theory-test-dataattribute-to-load-data-from-json-files/

[a comment by an automated account is not shown]

**Reply by andrewlock** [reply]
<em>[Sock](https://disqus.com/by/andrewlockdotnet/) (@andrewlock) commented [on Disqus](https://disqus.com/home/forum/andrewlock/) at November 16 2017, 08:22, in reply to [Scott](https://disqus.com/by/scott01011970/)'s [comment](https://github.com/andrewlock/blog-comments/discussions/237#discussioncomment-6043098)</em>

---

<p>Oops, my bad, forgot to push the code sorry! Link is working now :)</p>

[a comment by an automated account is not shown]

**Comment by marlonchosky** [comment]
Thanks for the implementation man!
